docs/content/latest/benchmark/tpcc-ysql/_index.md

```diff
@@ -63,7 +74,15 @@ Other options like username, password, port, etc. can be changed using the confi
 <password></password>
 ```
 
-## 3. Run the TPC-C benchmark
+## 3. Best practices
+
+**Latest TPCC code:** Use the latest enhancements to the Yugabyte TPCC application `git clone`.
```

Comment:
This is something we can avoid now. We are planning a release in this week. Once that is done, one can download the latest binaries from the release page.

Reply:
Thanks @hbhanawat! Going forward and similar to the past reason for this initially, the latest enhancements will always be found in the source itself. Recommend we keep git clone unless there is a reason not to?